api/src/main/java/org/pmiops/workbench/api/CohortAnnotationDefinitionController.java

```diff
@@ -48,10 +56,17 @@ public CohortAnnotationDefinition apply(org.pmiops.workbench.db.model.CohortAnno
             new Function<CohortAnnotationDefinition, org.pmiops.workbench.db.model.CohortAnnotationDefinition>() {
                 @Override
                 public org.pmiops.workbench.db.model.CohortAnnotationDefinition apply(CohortAnnotationDefinition cohortAnnotationDefinition) {
+                    List<String> enumValues = cohortAnnotationDefinition.getEnumValues();
+                    Set<CohortAnnotationEnumValue> enumValuesSet = (enumValues == null) ? null :
+                            new HashSet<CohortAnnotationEnumValue>(
+                                    IntStream.range(0, enumValues.size())
+                                            .mapToObj(i -> new CohortAnnotationEnumValue().name(enumValues.get(i)).order(i))
```

Comment:
How do you update an existing enum or enum value? Are these immutable once they are created?